### PR TITLE
RIA-3628: Remove Validation table headers fields for check

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3628: HO Validation tab alignment in single table, Stateless

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3628

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: e2e/features/submit-appeal.feature [hunk 1]
```diff
@@ -58,6 +55,3 @@ Feature: Submit appeal application
     And I should see the text `Review the appeal application form data and cross reference it with Home Office data. If the appeal looks valid, you must tell the respondent to supply their evidence.`
     And I should see the text `Request respondent evidence`
     And I should see the text `The first column of information is sourced from the appeal application form, the second column of information is from Home Office records`
-    And I should see the text `Appellant details`
-    And I should see the text `Application details`
```
Comment (jkumar-moj) [inline e2e/features/submit-appeal.feature:62]: These labels should still be there right? are the checks removed because they are inside a table?

Comment (divisathyan) [reply e2e/features/submit-appeal.feature:62]: Correct. E2E cannot see the fields inside table

**Review by jkumar-moj — DISMISSED** [review]

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Pull request merged**